src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Trumbowyg.Edit.cshtml

```diff
@@ -59,6 +59,19 @@ else
         </text>
     }
 }    
+
+    document.addEventListener('DOMContentLoaded', function() {
+        function setTheme() {
+            var theme = document.documentElement.dataset.bsTheme;
+            var isDark = theme === 'dark' || (theme === 'auto' && window.matchMedia('(prefers-color-scheme: dark)').matches);
```

Comment:
To prevent duplicating logic, add script dependency to a script names `admin-head` which is added globally already in the admin theme. Then using the function `isDarkMedia()` or `getPreferredTheme() == darkThemeName`

Reply:
Done